test/e2e/e2e_suite_test.go

```diff
@@ -160,6 +160,7 @@ func setupBootstrapCluster(config *clusterctl.E2EConfig, useExistingCluster bool
 	if !useExistingCluster {
 		clusterProvider = bootstrap.CreateKindBootstrapClusterAndLoadImages(context.TODO(), bootstrap.CreateKindBootstrapClusterAndLoadImagesInput{
 			Name:               config.ManagementClusterName,
+			KubernetesVersion:  "v1.27.3",
```

Comment:
can we add a note to revert this once CAPI fixes its own default version? Otherwise we might end up pinned to an old version later